### ISSUE TITLE
Suggestion: add "ember install ember-moment" to docs.

### ISSUE THREAD

**Issue by dirkdirk** [mydea/ember-date-components#92]
I see it on the github page but not here: http://mydea.github.io/ember-date-components/#/

**Comment by mydea** [comment]
Hmm, it says so in the text below the hero ("You need to ember install ember-moment separately for this add-on.") - I guess maybe that is not prominent enough?

**Comment by dirkdirk** [comment]
Thanks for the fast reply!

Ahh ... now I see it.  Here's what happened to me.  I saw the repo on Github, then went straight to the docs site (http://mydea.github.io/ember-date-components/#/), then ran $ ember install ember-date-components, then put {{date-picker ... }} into a template and got an error re. moment.  Wasn't till I looked back at the github page that I saw to install moment also.  

Mostly my bad.  But ya, making it more prominent might help other dingbats like me ... ;-) ... Perhaps put it above $ ember install ember-date-components in the "jumbo".
